go.mod

```diff
@@ -92,6 +92,7 @@ require (
 	google.golang.org/grpc v1.46.0 // indirect
 	google.golang.org/protobuf v1.28.0 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
+	gopkg.in/yaml.v3 v3.0.0 // indirect
```

Comment:
I don't like that we now have this in our go.mod.

Reply:
if it fixes the warning seems fine

Reply:
Let's see if it does.